floss/rust_version_database.json

```diff
@@ -0,0 +1 @@
af19a64603b02b4130fe52958cc12488900": "1.3.0", "082e4763615bdbe7b4dd3dfd6fc2210b7773edf5": "1.2.0", "35ceea3997c79a3b7562e89b462ab76af5b86b22": "1.1.0", "a59de37e99060162a2674e3ff45409ac73595c0e": "1.0.0", "522d09dfecbeca1595f25ac58c6d0178bbd21d7d": "1.0.0-alpha.2", "44a287e6eb22ec3c2a687fc156813577464017f7": "1.0.0-alpha", "ba4081a5a8573875fed17545846f6f6902c8ba8d": "0.12.0", "aa1163b92de7717eb7c5eba002b4012e0574a7fe": "0.11.0", "46867cc3e4ddcbb1d359a315805de00094dacaf9": "0.10", "7613b15fdbbb9bf770a2c731f4135886b0ff3cf0": "0.9", "8a4f0fa6c518eb634687abe9659601d9d2a61899": "0.8", "a2db7c15ce9f586164cabb15d83fb3f6bbeb3cf5": "0.7", "00dbbd01c2aee72982b3e0f9511ae1d4428c3ba9": "0.6", "8b98e5a296d95c5e832db0756828e5bec31c6f50": "0.5", "39c0d3591e0326874b7263a621ce09ecd64f0eb2": "0.4", "2f32a1581f522e524009138b33b1c7049ced668d": "0.3", "021bd294c039bd54aa5c4aa85bcdffb0d24bc892": "1.15.1", "10893a9a349cdd423f2490a6984acb5b3b7c8046": "1.15.0", "e8a0123241f0d397d39cd18fcc4e5e7edde22730": "1.14.0", "2c6933acc05c61e041be764cb1331f6281993f3f": "1.13.0", "d4f39402a0c2c2b94ec0375cd7f7f6d7918113cd": "1.12.1", "3191fbae9da539442351f883bdabcad0d72efcb6": "1.12.0", "9b21dcd6a89f38e8ceccb2ede8c9027cb409f6e3": "1.11.0", "cfcb716cf0961a7e3a4eceac828d94805cf8140b": "1.10.0", "e4e8b666850a763fdf1c3c2c142856ab51e32779": "1.9.0", "db2939409db26ab4904372c82492cd3488e4c44e": "1.8.0", "a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52": "1.64.0", "4b91a6ea7258a947e59c6522cd5898e7c0a6a88f": "1.63.0", "e092d0b6b43f2de967af0887873151bb1c0b18d3": "1.62.1", "a8314ef7d0ec7b75c336af2c9857bfaf43002bfc": "1.62.0", "fe5b13d681f25ee6474be29d748c65adcd91f69e": "1.61.0", "7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c": "1.60.0", "9d1b2106e23b1abd32fce1f17267604a5102f57a": "1.59.0", "db9d1b20bba1968c1ec1fc49616d4742c1725b4b": "1.58.1", "02072b482a8b5357f7fb5e5637444ae30e423c40": "1.58.0", "f1edd0429582dd29cccacaf50fd134b05593bd9c": "1.57.0", "fc50f328b0353b285421b8ff5d4100966387a997": "1.34.1", "91856ed52c58aa5ba66a015354d1cc69e9779bdf": "1.34.0", "2aa4c46cfdd726e97360c2734835aa3515e8c858": "1.33.0", "9fda7c2237db910e41d6a712e9a2139b352e558b": "1.32.0", "b6c32da9b0481e3e9d737153286b3ff8aa39a22c": "1.31.1", "abe02cefd6cd1916df62ad7dc80161bea50b72e8": "1.31.0", "1433507eba7d1a114e4c6f27ae0e1a74f60f20de": "1.30.1", "da5f414c2c0bfe5198934493f04c676e2b23ff2e": "1.30.0", "17a9dc7513b9fea883dc9505f09f97c63d1d601b": "1.29.2", "b801ae66425cf7c3c71052b19ef8f145b0d0513d": "1.29.1", "4d90ac38c0b61bb69470b61ea2cccea0df48d9e5": "1.24.0", "766bd11c8a3c019ca53febdcd77b2215379dd67d": "1.23.0", "05e2e1c41414e8fc73d0f267ea8dab1a3eeeaa99": "1.22.1", "328886ba2eaf0d3ac7e78ef3ba27eb296d0af3c0": "1.22.0", "3b72af97e42989b2fe104d8edbaee123cdf7c58f": "1.21.0", "f3d6973f41a7d1fb83029c9c0ceaf0f5d4fd7208": "1.20.0", "0ade339411587887bf01bcfa2e9ae4414c8900d4": "1.19.0", "03fc9d622e0ea26a3d37f5ab030737fcca6928b9": "1.18.0", "56124baa9e73f28c0709e59e74783cf234a978cf": "1.17.0", "30cf806ef8881c41821fbd43e5cf3699c5290c16": "1.16.0"}
```

Comment:
i think we'll need to update the pyinstaller config to reference this data file.

Reply:
we should also add a resource that explains where the file came from and how to regenerate this file.

Reply:
using a Python file is a fine idea. i think we should keep the file along the language identification file, so this is the appropriate place today. we can move it fairly easily.

Reply:
Yes, you can add the script to the `scripts` directory.